nir/src/main/scala/scala/scalanative/nir/Versions.scala

```diff
@@ -20,8 +20,8 @@ object Versions {
    * when 1.3-based release happens all of the code needs to recompiled with
    * new version of the toolchain.
    */
-  final val compat: Int   = java.lang.Integer.parseUnsignedInt("0")
-  final val revision: Int = java.lang.Integer.parseUnsignedInt("0")
+  final val compat: Int   = java.lang.Integer.parseUnsignedInt("1")
+  final val revision: Int = java.lang.Integer.parseUnsignedInt("1")
```

Comment:
Yeah, it's unlikely that we'll ever overflow numeric signed/unsigned boundary. Might as well be just `= 1`.